clap_builder/src/parser/matches/arg_matches.rs

```diff
@@ -1226,6 +1226,15 @@ impl ArgMatches {
         let presence = self.args.contains_key(id);
         Ok(presence)
     }
+
+    /// Clears current `ArgMatches` instance of all matches with the given id without returning them.
+    /// Doesn't require type specification thus can't fail because of type distinction.
+    /// Returns Err([`MatchesError`]) if the given id isn't valid for current `ArgMatches` instance.
+    /// Returns `Ok(true)` if there were any matches with the given id, `Ok(false)` otherwise.
```

Comment:
doc comments should have a one-line summary to start with

How about something like:
```suggestion
    /// Clears the values for the given `id`
    ///
    /// Alternative to [`try_remove_*`][ArgMatches::try_remove_one] when the type is not known.
    ///
    /// Returns `Err([`MatchesError`])` if the given `id` isn't valid for current `ArgMatches` instance.
    ///
    /// Returns `Ok(true)` if there were any matches with the given `id`, `Ok(false)` otherwise.
```

Reply:
Agreed, made it the way you suggested in f113f2cd97d5ae6960636ca41675f4e5c90e498e.